layouts/shortcodes/observability_pipelines/processors/set_env_vars.md

```diff
@@ -5,4 +5,26 @@ To set up this processor:
 1. Define a [filter query](#filter-query-syntax). Only logs that match the specified filter query are processed. All logs, regardless of whether they match the filter query, are sent to the next step in the pipeline.
 1. Enter the field name for the environment variable.
 1. Enter the environment variable name.
-1. Click **Add Environment Variable** if you want to add another environment variable.
\ No newline at end of file
+1. Click **Add Environment Variable** if you want to add another environment variable.
+
+##### Blocked environment variables
```

Comment:
```suggestion
##### Blocked environment variables
```

small nit but I noticed on this page there is no TOC, not sure if you wanted there to be one might consider changing this to H3 as well as the #filter-query-syntax section? Up to you though!

Reply:
Thanks @aliciascott for the note! There's some tech debt here that I haven't had a chance to clean up. The H3 works for the [main setup docs](https://docs-staging.datadoghq.com/may/security-access-control/security/access_control/), but not for these reference docs. I need to at some point pull out the headers into the doc so it's not in the shortcode, and then I make it different header numbers.